test/e2e/resources/components/scripts/ExtendedComponent.brs

```diff
@@ -1,4 +1,9 @@
 sub init()
     print "ExtendedComponent init"
     start()
+
+    print m.top.isSubtype("ExtendedComponent")
```

Comment:
Can we also add `m.top.parentSubtype` in here?

Reply:
sure